conf/server.properties

```diff
@@ -0,0 +1,50 @@
+#Minecraft server properties
+#Sat Sep 11 17:53:29 GMT 2021
+enable-jmx-monitoring=false
```

Comment:
what is `enable-jmx-monitoring`?

Reply:
It has something to do with java and beans.

> Exposes an MBean with the Object name net.minecraft.server:type=Server and two attributes averageTickTime and tickTimes exposing the tick times in milliseconds.

> In order for enabling JMX on the Java runtime you also need to add a couple of JVM flags to the startup as documented here. 
"here" -> https://docs.oracle.com/javase/8/docs/technotes/guides/management/agent.html
Not really sure what it does, but I don't think it's something worth digging into that much